src/gpu/save_and_compare_cpu_vs_gpu.c

```diff
@@ -39,6 +39,16 @@
 
 #define MAX(a, b) (((a) > (b)) ? (a) : (b))
 
+/************************************************************************
+
+helper functions for debugging
+
+please uncomment if needed...
+
+*************************************************************************/
+
+
+/*
```

Comment:
Because if someone adds a block comment within the block comment, you end up with strange things happening, since it will end at the end of the middle block.

I'm not sure if there's any benefit to highlighting commented code. You need to uncomment it to be sure it's really working anyway...